src/ray/object_manager/object_manager.cc

```diff
@@ -938,10 +938,12 @@ std::string ObjectManager::DebugString() const {
   result << "\n- num pull requests: " << pull_manager_->NumActiveRequests();
   result << "\n- num buffered profile events: " << profile_events_.size();
   result << "\n- num chunks received total: " << num_chunks_received_total_;
-  result << "\n- num chunks received total failed: " << num_chunks_received_total_failed_;
-  result << "\n- num chunks received cancelled: " << num_chunks_received_cancelled_;
-  result << "\n- num chunks received thrashed: " << num_chunks_received_thrashed_;
-  result << "\n- num chunks received, plasma error: "
+  result << "\n- num chunks received failed (all): " << num_chunks_received_total_failed_;
+  result << "\n- num chunks received failed / cancelled: "
+         << num_chunks_received_cancelled_;
+  result << "\n- num chunks received failed / thrashed: "
+         << num_chunks_received_thrashed_;
+  result << "\n- num chunks received failed / plasma error: "
```

Comment:
Addressing comment from other PR to clarify these counters are a subset of the above.